clang/lib/Driver/ToolChains/BareMetal.cpp

```diff
@@ -491,3 +491,26 @@ void baremetal::Linker::ConstructJob(Compilation &C, const JobAction &JA,
       JA, *this, ResponseFileSupport::AtFileCurCP(),
       Args.MakeArgString(TC.GetLinkerPath()), CmdArgs, Inputs, Output));
 }
+
+SanitizerMask BareMetal::getSupportedSanitizers() const {
+  const bool IsX86_64 = getTriple().getArch() == llvm::Triple::x86_64;
```

Comment:
Add a comment suggesting that this allows a superset of other targets allow so that it is clearer if someone intends to extend the list in the future (say. when we get a new sanitizer).

Reply:
thank you, done